mvg/mvg.py

```diff
@@ -257,17 +243,27 @@ def create_source(self, sid: str, meta: dict):
 
         meta : dict
             meta information
+
+        ignore_conflict : bool
+            Set to true to prevent exceptions for 409 Conflict errors
+            caused by trying to create an existing source. Defaults to False
         """
 
         logger.info("endpoint %s", self.endpoint)
         logger.info("creating source with source id=%s", sid)
         logger.info("metadata: %s", meta)
 
+        do_not_raise = []
+        if ignore_conflict:
+            do_not_raise.append(409)
```

Comment:
A bit more expressive and self-documenting, maybe?
```suggestion
            do_not_raise.append(requests.codes.CONFLICT) # 409
```

Reply:
I agree, very nice suggestion